### ISSUE TITLE
[SEMANTIC HTML]: Chatbot should be in a labelled region for screen reader navigation

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#25186]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a screen reader user, I like to navigate by landmarks, and would find it helpful to hear something unique about the chatbot, like "VA Virtual Agent, complementary".

## Details

The chatbot is in a div with `[role="complementary"]` that could use an aria-label so it has a unique name in screen readers' landmarks menu. Code snippet and screenshot below.

## Acceptance Criteria

- [ ] Chatbot region announces itself to screen readers with a unique name

## Environment

* https://staging.va.gov/virtual-agent/

## Proposed Solution (if known)

```diff
<div
+ aria-label="VA Virtual Agent"
  class="webchat--css-obatg-j7qwjs webchat--css-obatg-1egyv3b"
  role="complementary"
>
  ...
</div>
```

## WCAG or Vendor Guidance (optional)

* https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html

## Screenshots or Trace Logs

![Screen Shot 2021-05-24 at 4 47 37 PM](https://user-images.githubusercontent.com/934879/119412757-03ddcc00-bcb2-11eb-8ecf-b0b3ecd79c27.png)

**Comment by ianpsant** [comment]
Custom - Within our control and we are able to change. We can track this accessibility issue and address if PVA development moves forward towards production.

**Comment by ianpsant** [comment]
When the chatbot moves, we will need to reconsider this. For now closing.
